### DISCUSSION TITLE
FreeCad 0.21<title>

### DISCUSSION THREAD

**Discussion by marblemonster** [community/community#129525] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Maybe in wrong place, but freecad 0.21 has no tool editor. Can't create new tools. The paths generated are not complete.

**Comment by ebndev** [comment]
Hi @marblemonster, thanks for being a part of the GitHub Community!

You might want to try asking for help somewhere that focuses on this project, such as opening an issue in the project's repository or seeing if they have a support page on their website / social media.

We want to make sure you’re getting the best support you can and that your message goes where you intended, but this may not be the right place.

Best of luck!
